### ISSUE TITLE
Architect adds algorithm user manual

### ISSUE THREAD

**Issue by avidrucker** [avidrucker/autofocus-exp#47]
to supplemental docs, `Architect writes, adds "todo list recipe" algorithm.md`

WIP draft:
```
## User Steps:

1. Dot the first task on the list. You can do it straight away if you want to, but if you don’t want to you carry on scanning through the list dotting tasks that you intend to do.

2. At any stage you can either work on the task you have just dotted or carry on down the list dotting further tasks.

3. When you have worked on a task you cross it out and re-enter it at the end of the list if further work is needed on it.

4. You then can either work on the previous dotted task or carrying on dotting further tasks.

5. That’s basically it, but there are two rules dealing with the beginning and end of the list:
  1. Whenever you delete the first task on the list the new first task must be dotted
  2. Whenever you reach the end of the list you must work on the last dotted task on the list.

To sum up, except where rules 5.1 and 5.2 apply you always have a choice of:
- Working on the last dotted task on the list OR
- Adding further dots after the task you have just done.

Bonus: “Whenever a task is dotted which I am ready to do right now I stop scanning and do it. That’s all there is to it - it’s as simple as that, but the effect on the speed of the system is enormous.”
```

**Comment by avidrucker** [comment]
Architect: todo:
- [ ] Confirm that this content is backed up / directly referenced from an original Mark Forster source, otherwise close & mark "do not take action on this"
